source/includes/changelog/_19-11-0.md.erb

```diff
@@ -4,6 +4,10 @@
 
 - Introduced version 3 of the [Role Config API](#roles).
 - Introduced version 2 of the [Auth Config API](#authorization-configuration).
+- Introduced version 1 of the [Pipeline Instance API](#get-pipeline-instance).
```

Comment:
We are not introducing Pipeline Instance API in 19.11.0. It will be a part of 19.12.0